src/AvtaleSide/Oppgavelinje/OppgaveLenker.tsx

```diff
@@ -42,7 +42,7 @@ const OppgaveLenker: React.FunctionComponent = () => {
     }
     return (
         <>
-            {!arbeidstreningReadonly && (
+            {(!erArbeidstrening || (erArbeidstrening && !arbeidstreningReadonly)) && (
```

Comment:
Skal ikke noen oppgavelenker vises hvis det er arbeidstrening? 🤔

Reply:
Jo, men ikke hvis toggelen for read-only arbeidstrening er på.

Det denne sier er at vi skal alltid vise lenkene om det ikke er arbeidstrening, men vi skal kun vise lenkene dersom det er arbeidstrening og toggelen ikke er på.

Reply:
ja, ser det nå, ble forvirret av dobbel negatives her 😅

hadde kanskje vært enklere å lese med en sånn en linje på :42 isteden: 🤷‍♂️
`if (erArbeidstrening && !arbeidstreningReadonly) return null;`

Reply:
Ettersom denne skal fjernes "etter hvert" så tenker jeg at jeg ikke kommer til å bruke mye tid på at logikken skal være forståelig.

Kan heller ikke returnere `null` på linje 42 ettersom det fortsatt skal returneres andre ting fra denne komponenten hvis toggle er aktiv (varsellogg på linje 67)